framework/postcommit_tasks/handlers.py

```diff
@@ -0,0 +1,41 @@
+import logging
+import threading
+
+from website import settings
+
+_local = threading.local()
+logger = logging.getLogger(__name__)
+
+def postcommit_queue():
+    if not hasattr(_local, 'postcommit_queue'):
+        _local.postcommit_queue = []
+    return _local.postcommit_queue
+
+def postcommit_before_request():
+    _local.postcommit_queue = []
+
+def postcommit_after_request(response=None, base_status_error_code=500):
```

Comment:
Minor: `response` should probably be required, since you are checking `response.status_code` on the first line of the function.